### PR TITLE
Fix PHPUnit deprecations

### PR DESCRIPTION
**Pull request by fabpot**
#3103 followup for 2.x

### CODE REVIEW

File: tests/TemplateTest.php [hunk 1]
```diff
@@ -386,7 +383,7 @@ public function getGetAttributeTests()
 
     public function testGetIsMethods()
     {
-        $this->expectException('\Twig\Error\RuntimeError');
```
Comment (fabpot) [inline tests/TemplateTest.php:389]: Was a bad merge

File: tests/Cache/FilesystemTest.php [hunk 2]
```diff
@@ -81,7 +81,7 @@ public function testWrite()
 
     public function testWriteFailMkdir()
     {
-        $this->expectException('\RuntimeException');
+        $this->expectException(\RuntimeException::class);
```
Comment (jderusse) [inline tests/Cache/FilesystemTest.php:84]: Good catch, I had to stringify classes because of PHP 5.4. 
But then we removed it and I forgot to revert it

**Review by jderusse — APPROVED** [review]

**Comment by jderusse** [comment]
@fabpot next step is to add `: void` return type on `setup` and `teardown` methods and remove the travis env variable `SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT`

But I'm not sure it's something to do in this PR

**Comment by fabpot** [comment]
@jderusse I've now merged this PR. Can you do the next steps in another PR?

**Comment by jderusse** [comment]
I'm on it

**Pull request merged**